modules/eks/oidc.tf

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Remove empty lines from start of this file.

Reply:
Removed.  Missing descriptions added for the output variables.